### PR TITLE
Fix ios/android packaging pipeline failure

### PR DESCRIPTION
**Pull request by YUNQIUGUO**
### Description

Generated a new user api token for app center. Previous one is no longer valid.

Variable Group linked to the pipeline is also updated accordingly.

### Motivation and Context

Fix packaging pipeline

### CODE REVIEW

**Review by baijumeswani — APPROVED** [review]

File: tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml [hunk 1]
```diff
@@ -125,7 +125,7 @@ jobs:
         --test-series "master" \
         --locale "en_US" \
         --build-dir $(Build.BinariesDirectory)/app_center_test_mobile/ios_package_test/DerivedData/Build/Products/Debug-iphoneos \
-        --token $(app_center_api_token)
+        --token $(app_center_api_token_2)
```
Comment (edgchen1) [inline tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml:128]: can we just replace the old value of $(app_center_api_token)?

Comment (YUNQIUGUO) [reply tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml:128]: Sure, updated the old value. will leave this change open as no need for pipeline change required.

**Comment by snnn** [comment]
Sorry I was on vacation and didn't notice this PR. Is this PR still needed?

**Comment by YUNQIUGUO** [comment]
> Sorry I was on vacation and didn't notice this PR. Is this PR still needed?

Oh no worries, this change is actually not needed anymore. I can close it.

**Pull request closed without merging**